DIPs/DIP-35.md

```diff
@@ -0,0 +1,78 @@
+---
+DIP: 35
+Title: Make Devcon censorship resistant, privacy preserving and smarter
+Status: Draft
+Themes: Virtual experience, Freeform, Purchases & ID
+Tags: Software, Communications, Event Production, Event Operations
+Instances: Devcon 7 
+Authors: costanza@ethswarm.org
+Resources Required: Operations support
+Discussion: https://forum.devcon.org/t/devcon-2024-agenda-on-swarm-with-fair-data-society-and-brianknowsai/3357
+Created: 2024-02-21
+---
+
+## Summary of Proposal
+Hosting the Devcon website with the Agenda of DevCon on Swarm/FairData Society and integrating BrianKnowsAI. 
+This would make Devcon’ website decentralized, censorship resistant and privacy preserving, and would make Devcon’s digital efforts aligned with its values, while offering an web3 AI assistant to help attendees have an amazing experience__
+
+## Abstract
+Swarm is a decentralized storage and communication system that was initially incubated by the Ethereum Foundation.
+
+Fair Data Society is an initiative for decentralized self-sovereign data commons.
+
+BraianKnowsAI is a non-custodial AI assistant for performing transactions and search web3 info on a custom-built knowledge base, and a Swarm Foundation grantee.
+
+This DIP follows the Devcon 2022 Swarm DIP and enhances it:
+
+Hosting the Devcon website and agenda on Swarm/FairData Society and integrating it with BriankKnowsAI to have a web3 native AI assistant.
+
+## Motivation & Rationale
+
+Implementing this DIP would make Devcon’s digital efforts aligned with its principles and values:
+
+The website hosting on the Swarm Network will be web3 native, censorship resistant and privacy preserving. The website will be in control of the Devcon team and not of a centralized provider. 
+
+Adding BrianKnowsAI will enhance the experience for attendees and virtual visitors aslike, as attendees will be able to ask questions on the agenda, i.e. “Brian, when is Vitalik speaking?” or “Brian, what’s account abstraction?”. It will also help attendees perform transactions helpful for purchses "Brian bridge 100DAI2 from Ethereum Mainnet to Polygon", "Brian, transfer 10DAI to vitalik.eth to pay them for dinner"
```

Comment:
```suggestion
Adding BrianKnowsAI will enhance the experience for attendees and virtual visitors aslike, as attendees will be able to ask questions on the agenda, i.e. “Brian, when is Vitalik speaking?” or “Brian, what’s account abstraction?”. It will also help attendees perform transactions helpful for purchases "Brian bridge 100DAI2 from Ethereum Mainnet to Polygon", "Brian, transfer 10DAI to vitalik.eth to pay them for dinner"
```